packages/data-stores/src/plans/test/selectors.ts

```diff
@@ -0,0 +1,326 @@
+/**
+ * External dependencies
+ */
+import * as deprecate from '@wordpress/deprecated';
+
+/**
+ * Internal dependencies
+ */
+import * as Selectors from '../selectors';
+import * as MockData from '../mock';
+import { buildPlanFeaturesDict } from '../test-utils';
+import { TIMELESS_PLAN_ECOMMERCE, TIMELESS_PLAN_FREE, FREE_PLAN_PRODUCT_ID } from '../constants';
+import type { State } from '../reducer';
+
+// Test data
+const TEST_LOCALE_1 = 'test-locale-1';
+const TEST_LOCALE_2 = 'test-locale-2';
+
+const mockFeatures = {
+	[ TEST_LOCALE_1 ]: buildPlanFeaturesDict( [
+		MockData.STORE_PLAN_FEATURE_CUSTOM_DOMAIN,
+		MockData.STORE_PLAN_FEATURE_LIVE_SUPPORT,
+	] ),
+	[ TEST_LOCALE_2 ]: buildPlanFeaturesDict( [
+		MockData.STORE_PLAN_FEATURE_PRIORITY_SUPPORT,
+		MockData.STORE_PLAN_FEATURE_RECURRING_PAYMENTS,
+	] ),
+};
+const mockFeaturesByType = {
+	[ TEST_LOCALE_1 ]: [
+		MockData.API_FEATURES_BY_TYPE_GENERAL,
+		MockData.API_FEATURES_BY_TYPE_COMMERCE,
+	],
+	[ TEST_LOCALE_2 ]: [
+		MockData.API_FEATURES_BY_TYPE_GENERAL,
+		MockData.API_FEATURES_BY_TYPE_MARKETING,
+	],
+};
+const mockPlans = {
+	[ TEST_LOCALE_1 ]: [ MockData.STORE_PLAN_FREE, MockData.STORE_PLAN_PREMIUM ],
+	[ TEST_LOCALE_2 ]: [ MockData.STORE_PLAN_FREE ],
+};
+const mockPlanProducts = [ MockData.STORE_PRODUCT_FREE, MockData.STORE_PRODUCT_PREMIUM_ANNUALLY ];
+
+const mockState: State = {
+	features: mockFeatures,
+	featuresByType: mockFeaturesByType,
+	plans: mockPlans,
+	planProducts: mockPlanProducts,
+};
+
+// Mocks
+jest.mock( '@wordpress/deprecated', () => {
+	return jest.fn();
+} );
```

Comment:
Mocking `'@wordpress/deprecated'` because, later in the tests, we're going to check wether the function has been called, but without polluting the console while running the tests